src/v0/destinations/gainsight_px/transform.js

```diff
@@ -56,6 +60,16 @@ const identifyResponseBuilder = async (message, { Config }) => {
     payload.firstName = fName;
     payload.lastName = lName;
   }
+  // Only for the case of new user creation, if signUpDate is not provided in traits, timestamp / originalTimestamp is mapped
+  if (!isPresent && !payload.signUpDate) {
```

Comment:
what is Present is it userID or traits? make it more clearer

Reply:
at line number 54:   
`const { success: isPresent } = await objectExists(userId, Config, 'user');`

using the userID we are checking if the user is already present. and keeping the status in isPresent

Reply:
can we rename as isUserPresent?

Reply:
addressed